build.githubactions.ps1

```diff
@@ -128,11 +131,15 @@ function Compile {
 }
 
 function Pack {
-    if (-not $PackageName){
+    if ([string]::IsNullOrWhiteSpace($PackageName) -and [string]::IsNullOrWhiteSpace($NuspecFilePath)){
         Invoke-Execute {
             dotnet pack $ProjectFile -c $Configuration --output $packageOutput --no-build --verbosity normal -p:VersionPrefix=$version -p:NoWarn=NU5123
         }
-    } else {
+    }
+    if ($NuspecFilePath -Like "*.nuspec" -and $PackageName  -Like "EdFi.Suite3.Ods.Extensions.**"){   
+       nuget pack $NuspecFilePath -OutputDirectory $packageOutput -Version $version -Properties configuration=$Configuration -Properties id=$PackageName -NoPackageAnalysis -NoDefaultExcludes
+    }
+    if ([string]::IsNullOrWhiteSpace($NuspecFilePath) -and $PackageName  -Like "EdFi.Suite3.Ods.Extensions.**"){   
```

Comment:
Similar to the comment above this, should we instead just check that the NuspecFilePath is empty and then check that a package name has a value, and not that its a specific value, in case it needs to be extended further in the future?

Reply:
I added $PackageName -ne $null